specs/core/0_beacon-chain.md

```diff
@@ -800,7 +800,8 @@ def get_randao_mix(state: BeaconState,
     """
     Return the randao mix at a recent ``epoch``.
     """
-    assert get_current_epoch(state) - LATEST_RANDAO_MIXES_LENGTH < epoch <= get_current_epoch(state)
+    min_epoch = epoch - LATEST_RANDAO_MIXES_LENGTH + 1 if epoch > LATEST_RANDAO_MIXES_LENGTH else GENESIS_EPOCH
```

Comment:
I think it should be `get_current_epoch(state)` instead of `epoch`:
```suggestion
    min_epoch = get_current_epoch(state) - LATEST_RANDAO_MIXES_LENGTH + 1 if get_current_epoch(state) > LATEST_RANDAO_MIXES_LENGTH else GENESIS_EPOCH
```